editor/editor_run.cpp

```diff
@@ -71,25 +71,26 @@ Error EditorRun::run(const String &p_scene, const String &p_custom_args, const L
 	}
 
 	int screen = EditorSettings::get_singleton()->get("run/window_placement/screen");
-	if (screen == 0) {
-		// Same as editor
-		screen = DisplayServer::get_singleton()->window_get_current_screen();
-	} else if (screen == 1) {
-		// Previous monitor (wrap to the other end if needed)
-		screen = Math::wrapi(
-				DisplayServer::get_singleton()->window_get_current_screen() - 1,
-				0,
-				DisplayServer::get_singleton()->get_screen_count());
-	} else if (screen == 2) {
-		// Next monitor (wrap to the other end if needed)
-		screen = Math::wrapi(
-				DisplayServer::get_singleton()->window_get_current_screen() + 1,
-				0,
-				DisplayServer::get_singleton()->get_screen_count());
-	} else {
-		// Fixed monitor ID
-		// There are 3 special options, so decrement the option ID by 3 to get the monitor ID
-		screen -= 3;
+	switch (screen) {
+		case 0:
+			// Same as editor
+			screen = DisplayServer::get_singleton()->window_get_current_screen();
```

Comment:
You forgot `break` statements after each `case`. Otherwise, the `switch` statement will fall through.